index.html

```diff
@@ -160,6 +160,27 @@ <h4 class="card-paragraph">Skills</h4>
                 </div>
             </div>
         </section>
+        <section class="contact">
+            <h5>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h5>
+            <form class="my-form" action="https://formspree.io/f/xdobpeww" method="post">
+                <ul>
+                    <li><input type="text" name="fullname" id="username" maxlength="30" placeholder="Full Name" required></li>
+                    <li><input type="email" name="email" id="user_email" placeholder="Email Address" required></li>
+                    <li><textarea name="message" id="message" maxlength="500" cols="30" rows="10" required>Enter Text here</textarea></li>
+                </ul>
+                <button type="submit">Get in touch</button>
+            </form>
+        </section>
     </section>
+    <footer>
+        <ul class="social-media-links-footer">
+            <li><a href="https://twitter.com/belkaiduus" target="_blank" rel="noopener"><img rel="noreferrer" src="img/social/Twitter.png" alt="Twitter Logo" ></a></li>
+            <li><a href="https://linkedin.com/in/belkaidus" target="_blank" rel="noopener"><img rel="noreferrer" src="img/social/Linkedin.png" alt="Linkedin Logo" ></a></li>
+            <li><a href="https://medium.com/" target="_blank" rel="noopener"><img rel="noreferrer" src="img/social/medium.png" alt="Medium Logo" ></a></li>
+            <li><a href="https://github.com/mr-abdellah" target="_blank" rel="noopener"><img rel="noreferrer" src="img/social/GitHub.png" alt="GitHub Logo"></a></li>
+            <li><a href="https://angel.co/" target="_blank" rel="noopener"><img rel="noreferrer" src="img/social/angellist.png" alt="Angellist Logo"></a></li>
+        </ul>
+        <h6>belkaid.abdulah@gmail.com</h6>
```

Comment:
- [ ] Please kindly stick to the design guidelines(margins, padding, spacing etc) as much as possible. At the moment, your design does not match the original design

### Your Design
![image](https://user-images.githubusercontent.com/43995830/165333511-6fddf1e7-4e7d-44ed-8092-f2458449ef15.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/165333947-8ba6e1bf-1f1a-42be-86fd-f3b97de80ded.png)